### PR TITLE
[FIX] l10n_in_edi_ewaybill:  Deactivated E-Invoice (IN) edi before load demodata

### PR DESCRIPTION
**Pull request by sami-odoo**
Deactivated demo data in  l10n_in_edi_ewaybill module for demo E-Waybill Company

Runbot Nightly build give error: https://runbot.odoo.com/runbot/build/23714129

### CODE REVIEW

File: addons/l10n_in_edi_ewaybill/demo/chart_template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from odoo import models, api, Command
```
Comment (jva-odoo) [inline addons/l10n_in_edi_ewaybill/demo/chart_template.py:1]:
```suggestion
# Part of Odoo. See LICENSE file for full copyright and licensing details.

from odoo import models, api, Command
```

File: addons/l10n_in_edi_ewaybill/demo/chart_template.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    _inherit = 'account.chart.template'
+
+    @api.model
+    def _get_demo_data(self):
+        if self.env.company == self.env.ref('l10n_in_edi_ewaybill.demo_company_in_ewaybill'):
+            val = self.env['account.journal'].search([
+                ('type', '=', 'sale'),
+                ('company_id', '=', self.env.ref('l10n_in_edi_ewaybill.demo_company_in_ewaybill').id)])
+            a = {'edi_format_ids': [Command.unlink(self.env.ref('l10n_in_edi.edi_in_einvoice_json_1_03').id)]}
+            val.write(a)
```
Comment (jva-odoo) [inline addons/l10n_in_edi_ewaybill/demo/chart_template.py:14]:
```suggestion
            val.write({'edi_format_ids': [Command.unlink(self.env.ref('l10n_in_edi.edi_in_einvoice_json_1_03').id)]})
```

**Review by jva-odoo — CHANGES_REQUESTED** [review]
@sami-odoo check suggestions

**Review by jva-odoo — APPROVED** [review]

File: addons/l10n_in_edi_ewaybill/demo/chart_template.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api, Command
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    @api.model
+    def _get_demo_data(self):
```
Comment (jco-odoo) [inline addons/l10n_in_edi_ewaybill/demo/chart_template.py:10]:
```suggestion
    def _get_demo_data(self):
    """ We need to deactivate einvoice here, as we can not send e-invoice and e-waybill in the same demo company"""
```

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**